### DISCUSSION TITLE
Regarding power save configurations of ESP32-C6.

### DISCUSSION THREAD

**Discussion by Lalit2612** [espressif/arduino-esp32#9789] [Question - Community Help]
Hi,
I need to run power save example on ESP32-C6-DevKitC-1 board (with ESP32-C6-WROOM-1) in the light sleep mode. Can someone help me out with the memory configuration, clock configuration and peripheral configuration details while in sleep? In the datasheet, for the light sleep mode there are two configurations as follows:
![image](https://github.com/espressif/arduino-esp32/assets/82656243/b3957fb7-93f3-4611-88fc-b8b898080a05)
The second option shows that the peripherals are shut down. How to ensure that peripherals are shut down? Also, the documentation states that the SPI flash is not shut down by default in light sleep mode. How to power it down for minimal power consumption?

I am using Espressif IDE and SDK v5.2.1.

Thanks in advance,
Lalit.

**Comment by me-no-dev** [comment]
Hi @Lalit2612 . It seems you are looking for ESP-IDF and not Arduino? https://github.com/espressif/esp-idf

**Reply by Lalit2612** [reply]
Yes, I am not using arduino. I am using ESP-32 only in Espressif IDE.

**Reply by me-no-dev** [reply]
You can ask your question either in ESP-IDF's issue tracker or https://esp32.com

**Reply by Lalit2612** [reply]
Sure, thanks.
